### PR TITLE
Blog preview in homepage and blog styling

### PR DESCRIPTION
**Pull request by realaravinth**
- Show blog preview in homepage
- Improve blog listing styling
- responsive styling for posts

### CODE REVIEW

File: templates/members.html [hunk 1]
```diff
@@ -1,44 +1,53 @@
...
+      <li>
+        <a href="{{ members.records[record][5] }}" target="_blank"
+          ><img src="/icons/github.svg" alt="GitHub icon"
+        /></a>
+      </li>
+      {% endif %} {% if members.records[record][6] %}
+      <li>
+        <a href="{{ members.records[record][6] }}" target="_blank"
+          ><img src="/icons/instagram.svg" alt="Instagram icon"
+        /></a>
```
Comment (alan2000alex) [inline templates/members.html:37]: a couple of Instagram links seem to be broken.
Pooja, Graphic Designer.
Rakesh Dhruv, Tech Writer

Comment (realaravinth) [reply templates/members.html:37]: You manually checked all of them? :joy: 

cc @TheVoraciousReader

File: content/blog/Introduction-to-Open-Source.md [hunk 2]
```diff
@@ -15,7 +17,6 @@ The term originated with regards to software development to assign a specific ap
 Open source supports universal access to a product's design or blueprints via an open-source or libre license, as well as universal redistribution of that design or blueprint. The open-source model is a decentralized software-development model that encourages open collaboration.
 
 ## What is Open-Source?
-![OSC](/blog/blog1/img3.png)
```
Comment (alan2000alex) [inline content/blog/Introduction-to-Open-Source.md:33]: Nice, the images scale perfectly, now.

File: templates/blog.html [hunk 3]
```diff
@@ -5,20 +5,31 @@
...
+						<span>{{ page.description | safe }} </span>
+						
+                        <p class="link-list__meta">
+                          Posted {{  page.day }} 
+						  {{ macros::get_month(month=page.month) }},
+                          {{ page.year }} by {{ page.extra.author }} 
+						  - <b>{{ page.reading_time }} min read</b>
+                        </p>
                     </li>
-
```
[the remark this answers is not in the document]
Comment (realaravinth) [reply templates/blog.html:21]: mobile view or the regular one?

**Review by alan2000alex — APPROVED** [review]
Everything seems to be in working order. The page looks so much more appealing now. 
Great!
Just have a look at those contact links, and the post banner aesthetics.

File: templates/members.html (hunk 1 above)
Comment (TheVoraciousReader) [reply templates/members.html:37]: Pooja's link is wrong, that was my bad, i'll fix it. Also, @alan2000alex could you check with Dhruva's instagram link again ? I'm positive it's the one he provided.

**Comment by realaravinth** [comment]
Please ignore

@netlify /

File: templates/index.html [hunk 4]
```diff
@@ -31,6 +31,7 @@ <h2 class="main__group-content-title">Diversity in tech</h2>
           Womein in Open Source(WiOS) is committed to bringing diversity in
```
Comment (TheVoraciousReader) [inline templates/index.html:31]: typo here 👀

Comment (realaravinth) [reply templates/index.html]: good catch

**Pull request merged**